utils/obsproc/Rads2Ioda.h

```diff
@@ -0,0 +1,89 @@
+#ifndef GDAS_UTILS_OBSPROC_RADS2IODA_H_  // NOLINT
+#define GDAS_UTILS_OBSPROC_RADS2IODA_H_  // NOLINT
+
+#include <iostream>
+#include <netcdf>    // NOLINT (using C API)
+#include <string>
+
+#include "eckit/config/LocalConfiguration.h"
+
+#include <Eigen/Dense>    // NOLINT
+
+#include "ioda/Group.h"
+#include "ioda/ObsGroup.h"
+
+#include "NetCDFToIodaConverter.h"
+
+#include "oops/util/missingValues.h"
+
+namespace gdasapp {
+  class Rads2Ioda : public NetCDFToIodaConverter {
+   public:
+    explicit Rads2Ioda(const eckit::Configuration & fullConfig)
+    : NetCDFToIodaConverter(fullConfig) {
+    oops::Log::info() << this->windowBegin_ << std::endl;
```

Comment:
How about some strings that say 'window begin:', etc.

Reply:
ho ... in the log info @CoryMartin-NOAA ? Yes, good point.